### PR TITLE
Fix #207 and #208

### LINKED ISSUE #207: Update Openshift Vagrant file with updated template details and tag.
Current Vagrant file have old location of templates and it always uses master which need to updated since now openshift/origin have release tag. Currently latest tag is v1.1.1 and we have to add it.

### PR DESCRIPTION
**Pull request by praveenkumar**
- Add latest tag for openshift/origin
- Update templates
- Add insecure registry

### CODE REVIEW

File: components/centos/centos-openshift-setup/Vagrantfile [hunk 1]
```diff
@@ -61,6 +61,11 @@ Vagrant.configure(2) do |config|
   # config.landrush.guest_redirect_dns = false
 
   config.vm.provision "shell", inline: <<-SHELL
+    sed -i.back '/# INSECURE_REGISTRY=*/c\INSECURE_REGISTRY="--insecure-registry 172.30.0.0/16"' /etc/sysconfig/docker
+    systemctl restart docker.service
```
Comment (navidshaikh) [inline components/centos/centos-openshift-setup/Vagrantfile:65]: is the IP range for registry going to be same?

Comment (praveenkumar) [inline components/centos/centos-openshift-setup/Vagrantfile:65]: Yes it will be same.

File: components/centos/centos-openshift-setup/Vagrantfile [hunk 2]
```diff
@@ -177,12 +176,22 @@ Vagrant.configure(2) do |config|
...
       done
-      touch ${ORIGIN_DIR}/configured.templates
+      touch #{ORIGIN_DIR}/configured.templates
+    fi
+  SHELL
+
+  config.vm.provision "shell", inline: <<-SHELL
+    export KUBECONFIG=${OPENSHIFT_DIR}/admin.kubeconfig
+    if [ ! -f #{ORIGIN_DIR}/configured.templates ]; then
+      echo "[INFO] Install OpenShift templates ..."
```
Comment (navidshaikh) [inline components/centos/centos-openshift-setup/Vagrantfile]: typo
*Installing OpenShift templates..

**Comment by LalatenduMohanty** [comment]
@praveenkumar LGTM. However you should squash the commits in to one as these looks like a single commit change.

**Pull request merged**